rules/windows/defense_evasion_execution_msbuild_started_by_script.toml

```diff
@@ -12,40 +12,40 @@ behavior is unusual and is sometimes used by malicious payloads.
 false_positives = ["The Build Engine is commonly used by Windows developers but use by non-engineers is unusual."]
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License v2"
 name = "Microsoft Build Engine Started by a Script Process"
 risk_score = 21
 rule_id = "9d110cb3-5f4b-4c9a-b9f5-53f0a1707ae2"
 severity = "low"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type: start and
-  process.name:MSBuild.exe and
-  process.parent.name:(cmd.exe or powershell.exe or cscript.exe or wscript.exe)
+process where event.type == "start" and
+  (process.name : "MSBuild.exe" or process.pe.original_file_name == "MSBuild.exe") and
```

Comment:
downside of adding the original file name to the rule for EQL is that now it's a requirement

Reply:
there is an OR process.name or ofn